cap-primitives/src/rustix/fs/metadata_ext.rs

```diff
@@ -228,7 +228,11 @@ impl MetadataExt {
             permissions: PermissionsExt::from_raw_mode(RawMode::from(statx.stx_mode)),
             modified: system_time_from_rustix(statx.stx_mtime.tv_sec, statx.stx_mtime.tv_nsec as _),
             accessed: system_time_from_rustix(statx.stx_atime.tv_sec, statx.stx_atime.tv_nsec as _),
-            created: system_time_from_rustix(statx.stx_btime.tv_sec, statx.stx_btime.tv_nsec as _),
+            created: if statx.stx_mask & StatxFlags::BTIME.bits() != 0 {
```

Comment:
The corresponding check in `std` is currently here: https://github.com/rust-lang/rust/blob/master/library/std/src/sys/unix/fs.rs#L461
I don't think it's necessary to check the mask for other fields that have been around for a long time, but I'm not sure.

Reply:
It looks like [std does the checks](https://github.com/rust-lang/rust/blob/master/library/std/src/sys/unix/fs.rs#L333) for `STATX_MTIME`, `STATX_ATIME`, and `STATX_CTIME`, so I think cap-std should too.

Reply:
Sorry for the delay on my end. I don't know why that's guarded by `#[cfg(target_pointer_width = "32")]` in std, but it seems harmless to just always check it. I'll update this.